CLOUDP-323936 - re-add securityContext settings to HelmCharts (#500)
mongodb/mongodb-kubernetes: changelog/20251007_feature_reintroduce_securitycontext_operator_setting_in.md

---
kind: feature
date: 2025-10-07
---

> * **Helm Chart**: Reintroduce a way to configure `securityContext` for Operator deployment through Helm Chart. New fields are `operator.podSecurityContext` and `operator.securityContext`
>  

Reviewer: nit:
```suggestion
* **Helm Chart**: Introduced two new helm fields `operator.podSecurityContext` and `operator.securityContext` that can be used to configure `securityContext` for Operator deployment through Helm Chart. 
```
what do you think about this?
